### DISCUSSION TITLE
Extract SD card from turtlebot4

### DISCUSSION THREAD

**Discussion by Alice-devpe** [turtlebot/turtlebot4#375] [Q&A]
I need to flash the sd card to upgrade ros2 from galactic y humble, but i'm trying to extract the sd card from turtlebot4 (not lite version) but in don't get how to take out the raspberry, is there any tutorial?

**Comment by hilary-luo** [comment]
If you remove the PCBA then you should be able to access the pi and SD card. https://turtlebot.github.io/turtlebot4-user-manual/mechanical/turtlebot4.html#removing-the-pcba
